pkg/daemon/testdata/synce-profile.json

```diff
@@ -0,0 +1,120 @@
+			  "-v",
+			  "1",
+			  "-e",
+			  "SURVEYIN,600,50000"
+			],
+			"reportOutput": true
+		  },
+		  {
+			"args": [
+			  "-P",
+			  "29.20",
+			  "-p",
+			  "MON-HW"
+			],
+			"reportOutput": true
+		  },
+		  {
+			"args": [
+			  "-P",
+			  "29.20",
+			  "-p",
+			  "CFG-MSG,1,38,300"
+			],
+			"reportOutput": true
+		  }
+		]
+	  }
+	},
+	"ptp4lConf": "[ens7f0]\nmasterOnly 1\n[ens7f1]\nmasterOnly 1\n[global]\n#\n# Default Data Set\n#\ntwoStepFlag 1\npriority1 128\npriority2 128\ndomainNumber 24\n#utc_offset 37\nclockClass 6\nclockAccuracy 0x27\noffsetScaledLogVariance 0xFFFF\nfree_running 0\nfreq_est_interval 1\ndscp_event 0\ndscp_general 0\ndataset_comparison G.8275.x\nG.8275.defaultDS.localPriority 128\n#\n# Port Data Set\n#\nlogAnnounceInterval -3\nlogSyncInterval -4\nlogMinDelayReqInterval -4\nlogMinPdelayReqInterval 0\nannounceReceiptTimeout 3\nsyncReceiptTimeout 0\ndelayAsymmetry 0\nfault_reset_interval 4\nneighborPropDelayThresh 20000000\nmasterOnly 0\nG.8275.portDS.localPriority 128\n#\n# Run time options\n#\nassume_two_step 0\nlogging_level 6\npath_trace_enabled 0\nfollow_up_info 0\nhybrid_e2e 0\ninhibit_multicast_service 0\nnet_sync_monitor 0\ntc_spanning_tree 0\ntx_timestamp_timeout 50\nunicast_listen 0\nunicast_master_table 0\nunicast_req_duration 3600\nuse_syslog 1\nverbose 0\nsummary_interval -4\nkernel_leap 1\ncheck_fup_sync 0\n#\n# Servo Options\n#\npi_proportional_const 0.0\npi_integral_const 0.0\npi_proportional_scale 0.0\npi_proportional_exponent -0.3\npi_proportional_norm_max 0.7\npi_integral_scale 0.0\npi_integral_exponent 0.4\npi_integral_norm_max 0.3\nstep_threshold 0.0\nfirst_step_threshold 0.00002\nclock_servo pi\nsanity_freq_limit  200000000\nntpshm_segment 0\n#\n# Transport options\n#\ntransportSpecific 0x0\nptp_dst_mac 01:1B:19:00:00:00\np2p_dst_mac 01:80:C2:00:00:0E\nudp_ttl 1\nudp6_scope 0x0E\nuds_address /var/run/ptp4l\n#\n# Default interface options\n#\nclock_type BC\nnetwork_transport L2\ndelay_mechanism E2E\ntime_stamping hardware\ntsproc_mode filter\ndelay_filter moving_median\ndelay_filter_length 10\negressLatency 0\ningressLatency 0\nboundary_clock_jbod 0\n#\n# Clock description\n#\nproductDescription ;;\nrevisionData ;;\nmanufacturerIdentity 00:00:00\n userDescription ;\ntimeSource 0x20\n",
+	"ptp4lOpts": "-2 --summary_interval -4",
+	"ptpSchedulingPolicy": "SCHED_FIFO",
+	"ptpSchedulingPriority": 10,
+	"synce4lConf": "[global]\nlogging_level              7\nuse_syslog                 0\nverbose                    1\nmessage_tag                [synce4l]\n[<synce1>]\nnetwork_option             2\nextended_tlv               1\nrecover_time               60\n#clock_id 5799633565433967748\nmodule_name                ice\ndnu_prio 0XFF\n[ens7f0]\ntx_heartbeat_msec          1000\nrx_heartbeat_msec          10\nallowed_qls                0xE\nallowed_ext_qls            0x40\n[{GNSS}]\nboard_label                GNSS-1PPS\ninput_QL                   0x1\ninput_ext_QL               0x20\n",
```

Comment:
Is it possible to have this use linebreaks so it shows up as multiple lined in this file?  Would be cleaner if we need to do single line changes in the future.

Reply:
Unfortunately JSON doesn't allow real line breaks in the string data... But I like the idea - will try with yaml

Reply:
done